bench_test.go

```diff
@@ -122,3 +122,23 @@ func BenchmarkDecimalString(b *testing.B) {
 		_ = corpus[rng.Intn(len(corpus))].String()
 	}
 }
+
+func BenchmarkSetFloat(b *testing.B) {
```

Comment:
nit: `BenchmarkDecimalSetFloat` because this is a method on `Decimal`.

Reply:
Done.